autogpt/commands/email.py

```diff
@@ -0,0 +1,32 @@
+import os
+from dotenv import load_dotenv
+import smtplib
+from email.message import EmailMessage
+
+load_dotenv()
```

Comment:
better initialize .env variables in config.py

Reply:
i created a pull request 2 days ago to add email feature . iam suggesting all this changes from that.

Reply:
thx done

Reply:
Let's get either merged, I briefly looked at yours, and you have many superfluous files in the files changed. Plus some other stuff. Would be good to have them more aligned.